src/styles/_card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  &__image {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    color: $secondary-color;
+    line-height: 14px;
+    font-weight: 400;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    margin-bottom: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-text {
+      font-weight: 400;
+      color: $secondary-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: bold;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    height: 40px;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $blue-accent;
+    transition: all 0.3s;
+
+    &:hover {
+      background-color: #fff;
```

Comment:
The `background-color` property in the `&:hover` state is set to `#fff`, which is the same as the card's background color. This might make the button blend with the card background when hovered. Consider using a different color for better visibility.

Reply:
button gain a border during hover effect